cpg_infra/abstraction/azure.py

```diff
@@ -48,9 +48,9 @@ def __init__(
         self.storage_account_lifecycle_rules = []
         self.storage_account_undelete_rule = None
 
-        data = az.authorization.get_client_config()
-        self.subscription_id = '/subscriptions/' + data.subscription_id
-        self.tenant = data.tenant_id
+        # data = az.authorization.get_client_config()
```

Comment:
This seems unrelated?

Reply:
This was initially failing with some related az auth issue, but then it occurred through this migration process that it's not needed. I've removed the line instead of commenting it out.